tests/sentry/api/endpoints/notifications/test_notification_actions_available.py

```diff
@@ -40,7 +40,7 @@ class MockActionRegistration(ActionRegistration):
             serialize_available = MagicMock(return_value=[trigger_available_response])
 
         registration = MockActionRegistration
-        NotificationAction.register_trigger_type(*trigger)
+        NotificationAction._trigger_types += (trigger,)
```

Comment:
this is never undone (test pollution)

I think this test can just be deleted -- afaik it was only there to demonstrate the dynamic behaviour

Reply:
good catch, i'll address it in the last PR since https://github.com/getsentry/getsentry/pull/12817 is merged